torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -21,6 +23,7 @@
 from ._fsdp_common import FSDPMeshInfo, HSDPMeshInfo, TrainingState
 from ._fsdp_param import FSDPParam, ParamModuleInfo, ShardedState
 
+logger = logging.getLogger("torch.distributed._composable.fsdp")
```

Comment:
Could you explain the reason for why we pass this string explicitly instead of `__name__`? I might have missed the context.

Reply:
the log level is controlled by the user at runtime with` TORCH_LOGS `or `torch._logging.set_logs.` The `__name__ `is used to ensure messages are routed to the correct location. The logging system is hierarchical, so if you setup logging for a module A, any submodules A.B, or A.C.D, will be handled by the infra for A. In this case, we setup logging for `torch.distributed._composable.fsdp` so any submodules will route through the handling of this module.